db/migrate/20220428203334_add_last_refreshed_to_katello_alternate_content_sources.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastRefreshedToKatelloAlternateContentSources < ActiveRecord::Migration[6.0]
+  def change
+    add_column :katello_alternate_content_sources, :last_refreshed, :datetime
+  end
+end
```

Comment:
Wondering if this makes more sense on the AlternateContentSmartProxy object rather than on ACS itself since the task is ACS-smartproxy related.

Reply:
I don't think having per-Pulp-ACS refresh time would be helpful, since we can't display it in the UI or API really.  I'd like to keep the idea of Katello ACSs as one whole unit so it's a bit easier for the user.  So, the last refreshed time would just be when Katello finished coordinating n Pulp ACS refreshes.